### PR TITLE
test/Gemfile: Add test-unit

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

File: Library/Homebrew/test/Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 gem "mocha", "~> 1.1"
 gem "minitest", "~> 5.3"
 gem "rake", "~> 10.3"
+gem "test-unit", "~> 3.2"
```
Comment (bw-yuhogg) [inline Library/Homebrew/test/Gemfile:6]: :+1:, but we'll have to remember to update this periodically unless we send this patch upstream.

Comment (sjackman) [reply Library/Homebrew/test/Gemfile:6]: We should get a merge conflict when upstream bumps the other versions, at which point we can bump `test-unit`.

**Comment by sjackman** [comment]
This fix might eliminate the need for the hack added in d2bbafa9043e420ded794515d128e880fdbbdec4

**Comment by sjackman** [comment]
Nope. That test still fails with `Error: Testing requires the latest version of testball_copy`

**Comment by sjackman** [comment]
Merged.

**Pull request closed without merging**